bigbluebutton-html5/imports/ui/components/actions-bar/mute-button/component.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {createContainer} from 'meteor/react-meteor-data';
+import {callServer} from '/imports/ui/services/api';
+import Button from '/imports/ui/components/button/component';
+import Users from '/imports/api/users/index';
+import Auth from '/imports/ui/services/auth/index';
+
+export default class MuteAudioComponent extends React.Component {
+  muteUser() {
+    callServer('muteUser', Auth.userID);
```

Comment:
You need to pass those callbacks as props from the container/service. The component shouldn't use `callSever` functions. You can take look into the Chat service, container and component.

https://github.com/bigbluebutton/bigbluebutton/blob/e26e9fd06029ea92c7b19769e4e532c2bf894eda/bigbluebutton-html5/imports/ui/components/chat/container.jsx#L82-L92